Drop-In/src/test/java/com/braintreepayments/api/dropin/DropInActivityUnitTest.java

```diff
@@ -566,6 +566,22 @@ public void onPaymentMethodNoncesUpdated_withNoCards_doesNotSendAnalyticEvent()
         verify(mActivity.braintreeFragment, never()).sendAnalyticsEvent("vaulted-card.appear");
     }
 
+    @Test
+    public void onPaymentMethodNoncesUpdated_withPayPal_doesNotSendCardAnalyticEvent() throws JSONException {
```

Comment:
I think this needs to be changed to `onActivityResult_withPayPal_doesNotSendCardAnalyticEvent`.